src/main/java/core/basesyntax/FiguresGenerator.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FiguresGenerator {
+    private Random random = new Random();
+    
+    public Figure[] generateFiguresArray() {
+        Figure[] figures = new Figure[random.nextInt(100)];
+        for (int i = 0; i < figures.length; i++) {
+            figures[i] = generateFigure();
+        }
+        return figures;
+    }
+    
+    private Figure generateFigure() {
+        switch (random.nextInt(5)) {
```

Comment:
declare 5 as a constant, now it's magic number
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names